readme.txt

```diff
@@ -53,3 +53,6 @@ Usage:
   grunt-wp-readme-to-markdown ^2.0.1, and update plugin version in `package.json`.
 * Upgrade Docker environment to use `wordpress:5.4.1-php7.4` image.
 * Correct contributor list in `readme.txt` and update `README.md`.
+
+= 0.1.3 (September 16th, 2020) =
+* Check for failures and errors on the error report page: https://github.com/WordPress/phpunit-test-reporter/pull/84
```

Comment:
Maybe "Include errors along with failures on the error report page"?
Or some other way it makes it clear they are being added (or were excluded before).

This is minor, but I'm wondering how the link will look written out like that if it were on a plugin page.
Maybe turn it into a formatted link with (PR) or something as the text if we want to link to the PR?

(like Include errors along with failures on the error report page ([PR](https://github.com/WordPress/phpunit-test-reporter/pull/84)).)